samples/python-v2-azurefunctions/function_app.py

```diff
@@ -0,0 +1,125 @@
+# import logging
+# import json
+# import azure.functions as func
+
+
+# def main(payload,
+#          order: func.Out[str]) -> None:
+#     logging.info(
+#         'Python function processed a CreateNewOrder request from the Dapr Runtime.')
+#     payload_json = json.loads(payload)
+#     logging.info(payload_json["data"])
+#     order.set(json.dumps({"value": payload_json["data"]}))
+
+
+import json
+import azure.functions as func
+import logging
+import json
+import azure.functions as func
+import logging
+from dapr_blueprint_app import app
+
+dapp = func.DaprFunctionApp()
+dapp.register_blueprint(app)
+
+# Dapr state output binding with http trigger
+@dapp.function_name(name="HttpTriggerFunc")
+@dapp.route(route="req", auth_level=dapp.auth_level.ANONYMOUS)
+@dapp.dapr_state_output(arg_name="state", state_store="statestore", key="newOrder")
+def main(req: func.HttpRequest, state: func.Out[str] ) -> str:
+    # request body must be passed this way '{\"value\": { \"key\": \"some value\" } }'
+    body = req.get_body()
+    logging.info(body.decode('utf-8'))
+    if body is not None:
+        state.set(body.decode('utf-8'))
+        logging.info(body.decode('utf-8'))
+    else:
+        logging.info('req body is none')
+    return 'ok'
+
+# Dapr state output binding with http dapr_service_invocation_trigger
+@dapp.function_name(name="CreateNewOrder")
+@dapp.dapr_service_invocation_trigger(arg_name="payload", method_name="CreateNewOrder")
+@dapp.dapr_state_output(arg_name="state", state_store="statestore", key="order")
+def main(payload: str, state: func.Out[str] ) :
+    # request body must be passed this way '{\"value\": { \"key\": \"some value\" } }'
+    logging.info('Python function processed a CreateNewOrder request from the Dapr Runtime.')
+    logging.info(payload)
+    if payload is not None:
+        state.set(payload)
+    else:
+        logging.info('payload is none')
+
+# Dapr state input binding with http dapr_service_invocation_trigger
+@dapp.function_name(name="RetrieveOrder")
+@dapp.dapr_service_invocation_trigger(arg_name="payload", method_name="RetrieveOrder")
+@dapp.dapr_state_input(arg_name="data", state_store="statestore", key="order")
```

Comment:
Shall we either use %StateStoreName% and parameterized values everywhere, or remove them completely for consistency?

Reply:
We will go with %StateStoreName% convention.